README.md

```diff
@@ -97,3 +97,13 @@ compiler version is to add the following fields to your VSCode user settings:
 ```
 
 Where of course `v0.8.4+commit.c7e474f2` can be replaced with any other version.
+
+## Contract verification
```

Comment:
can you remove the ComplexNRewarder stuff from this PR?

Reply:
rebased on top of main sadly, hard to now 
F

Reply:
b/sjoe-transfer branch I'm merging into should rebase on top of main to get rid of this diff